rust/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/zsh
+
```

Comment:
would appreciate some quick comments on expected usage as well - i see `DEFAULT_SIGNER_KEY` in the compose but can't see where we're configuring it

Reply:
ah just saw your slack message - ignore this